relay-event-normalization/src/normalize/span/description/resource.rs

```diff
@@ -0,0 +1,57 @@
+//! Code for scrubbing resource span description.
+use std::collections::BTreeSet;
+
+use once_cell::sync::Lazy;
+
+// TODO: move functions from parent module here.
```

Comment:
I will move resource-related functions in a follow-up PR, so this one retains a readable diff.